sdk/state/asset.go

```diff
@@ -41,3 +41,10 @@ func (a Asset) StringCanonical() string {
 	}
 	return fmt.Sprintf("%s:%s", a.Code(), a.Issuer())
 }
+
+func (a Asset) EqualsTrustLineAsset(ta xdr.TrustLineAsset) bool {
```

Comment:
💡 To follow convention of not making equal plural. This is my fault sorry, I suggested equals before.
```suggestion
func (a Asset) EqualTrustLineAsset(ta xdr.TrustLineAsset) bool {
```